client/dashboard/data/domain-forwarding.ts

```diff
@@ -26,3 +26,20 @@ export function deleteDomainForwarding(
 		domain_redirect_id: forwardingId,
 	} );
 }
+
+export interface DomainForwardingFormData {
```

Comment:
We shouldn't have any concept of "form" in the data layer. Let's use the `save` terminology since it's what being used everywhere else here.

```suggestion
export interface DomainForwardingSaveData {
```